build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -149,7 +149,7 @@ def _convert_flags_block(self, flags):
     return "  FLAGS\n%s\n" % (flags_list)
 
   def _convert_translate_tool_block(self, translate_tool):
-    if translate_tool:
+    if translate_tool and translate_tool != "//iree/tools:iree-translate":
```

Comment:
Hmmm I don't love this way of handling the defaults. Now it's coded in 4 places, but ok.

Reply:
I don't love the way that it was always generating the default value more ;)

Reply:
Yeah fair. I'd like to make this whole thing more robust, but it's hard to justify when it's currently working.